clang-tools-extra/test/clang-tidy/checkers/misc/bool-bitwise-operation-ignore-macros.cpp

```diff
@@ -0,0 +1,59 @@
+// RUN: %check_clang_tidy %s misc-bool-bitwise-operation %t \
+// RUN:   -config="{CheckOptions: { \
+// RUN:     misc-bool-bitwise-operation.IgnoreMacros: true }}"
+
+#define MY_OR |
+#define MY_AND &
+#define MY_OR_ASSIGN |=
+#define MY_AND_ASSIGN &=
+#define MY_LOG_AND &&
```

Comment:
Please add tests where you use these

Reply:
The test you've asked for will become not actual in future versions(See note https://github.com/denzor200/llvm-project/blob/b9be35d4d4bec7ede2ebe141eeb8a9d4fe1c550c/clang-tools-extra/test/clang-tidy/checkers/misc/bool-bitwise-operation.cpp#L352 )
I will just remove these defines